### PR TITLE
Add more code highlighting in README

### PR DESCRIPTION
**Pull request by bryant1410**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ To run the demo locally for development, you will need to:
 
 1. Create a fresh environment:
 
-    ```
-    conda create -n allennlp-demo python=3.6
-    source activate allennlp-demo
+    ```bash
+    conda create -n allennlp-demo python=3.7
+    conda activate allennlp-demo
```
Comment (matt-gardner) [inline README.md]: @schmmd, I think you said something recently about using `source` vs. `conda` here - do you have an opinion on this change?

Comment (bryant1410) [reply README.md]: (just as a reference: https://stackoverflow.com/a/49600711/1165181)

Comment (schmmd) [reply README.md]: What I said only applied to Slurm jobs.  `conda` is preferred in general, but it has some weird integrations with your environment so it doesn't work when running a job.

This change seems great to me!

**Comment by schmmd** [comment]
Thanks @bryant1410 !

File: README.md (hunk 1 above)
Comment (bryant1410) [reply README.md]: Oh, yeah, I think you're referring to conda/conda#7980. [You can also do the following](https://github.com/conda/conda/issues/7980#issuecomment-524154596):

```bash
eval "$(conda shell.bash hook)"
conda activate my_env
```

**Pull request merged**